{"name": "test_mul[zero]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0 == 1\n  +0\n  -1", "trace": "self = <testcases.test_cacu_yaml.TestCalc object at 0x000002230D8F6E88>, setprint = None, a = 1, b = 0, expect = 1\n\n    @pytest.mark.run(order=3)\n    @pytest.mark.parametrize('a,b,expect', datas['mul']['datas'], ids=datas['mul']['ids'])\n    def test_mul(self, setprint, a, b, expect):\n        # calc = Calculator()\n        result = self.calc.mul(a, b)\n>       assert round(result, 2) == expect\nE       assert 0 == 1\nE         +0\nE         -1\n\ntest_cacu_yaml.py:46: AssertionError"}, "parameters": [{"name": "a", "value": "1"}, {"name": "b", "value": "0"}, {"name": "expect", "value": "1"}], "start": 1604643590116, "stop": 1604643590116, "uuid": "5272f507-25a2-47fb-922f-c54289d8aba0", "historyId": "a0ff39358cdfeeb0e77f443f03a74433", "testCaseId": "329e25eee09c3181dbdbc6233c1cee55", "fullName": "test_cacu_yaml.TestCalc#test_mul", "labels": [{"name": "tag", "value": "run(order=3)"}, {"name": "suite", "value": "test_cacu_yaml"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "DESKTOP-FCPOG81"}, {"name": "thread", "value": "14904-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cacu_yaml"}]}